pkg/assetfetcher/http.go

```diff
@@ -52,6 +51,10 @@ type NexposeAssetFetcher struct {
 	HTTPClient *http.Client
 	// The Nexpose host that points to your instance
 	Host string
+	//
```

Comment:
question: did you intent to leave comments here?

Reply:
Yep, sure did. Thanks for pointing that out.